docker/ds/bootstrap/example-v1.json

```diff
@@ -14,7 +14,7 @@
             "subResources": {
                 "users": {
                     "type": "collection",
-                    "dnTemplate": "ou=People,o=userstore",
+                    "dnTemplate": "ou=People,dc=data",
```

Comment:
shouldn't it be possible to use @BASE_DN@ ?

Reply:
it's on the list of things to try to do later, there are lot of places where suffix i hardwired.